doc/tutorials/create-up2-images.sh

```diff
@@ -46,7 +46,7 @@ create_sos_images() {
         return 1
     }
 
-    SOS_BOOTARGS_DEBUG=sos_rootfs/usr/share/acrn/samples/up2/sos_bootargs_debug.txt
+    SOS_BOOTARGS_DEBUG=sos_rootfs/usr/share/acrn/samples/apl-up2/sos_bootargs_debug.txt
```

Comment:
I asked about this in https://github.com/projectacrn/acrn-hypervisor/pull/2667 and was told the sample folder names weren't changing?

Reply:
> I asked about this in #2667 and was told the sample folder names weren't changing?

PR#2761 merged which has modified the up2 name to apl-up2

Reply:
Ha. It looks like the folder name did change later. OK, thanks.